src/routing/RouterConfig.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
```suggestion
import type { FC } from 'react';
```


参考: https://qiita.com/Yz_4230/items/ce70927d36f7a6cf619d